ui/app/AppLayouts/Profile/views/NotificationsView.qml

```diff
@@ -23,16 +21,10 @@ import "../popups"
 
 SettingsContentBase {
     id: root
+    property bool hasMultipleDevices: true
```

Comment:
Do we really need this property?
We also have here the `deviceStore` so we can directly keep the binding `visible: root.store.devicesStore.devicesModel.count > 0` in `contentColumn`

Reply:
yes because the component isn't supose to care about the details (how this is calculated), so it's used as a property that the caller should specify